server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/GroupMappingTest.java

```diff
@@ -434,7 +434,48 @@ public void messageShouldRedirectToGroupWhenDomainMapping() throws Exception {
     }
 
     @Test
-    public void externalGroupMemberAreNotSupported() throws Exception {
+    public void messageShouldRedirectToGroupContainingSlash() throws Exception {
+        String groupWithSlash = "a/a@" + DOMAIN1;
+        String groupWithEncodedSlash = "a%2Fa@" + DOMAIN1;
```

Comment:
note that `@` is also encoded in URL so you're using incorrect test data

Reply:
ok I added an integration test for @ encoding

Reply:
ok